src/Umbraco.Core/Services/Implement/EntityXmlSerializer.cs

```diff
@@ -447,12 +448,17 @@ public XElement Serialize(IContentType contentType)
 
             var tabs = new XElement("Tabs", SerializePropertyGroups(contentType.PropertyGroups)); // TODO Rename to PropertyGroups
 
+       
+
             var xml = new XElement("DocumentType",
                 info,
                 structure,
                 genericProperties,
                 tabs);
 
+            // Never null in v8, ContentTypeCommonRepository.MapHistoryCleanup always returns an instance even if no db record.
+            xml.Add(SerializeCleanupPolicy(contentType.HistoryCleanup));
```

Comment:
The entity might not come from the repository, so lets be safe and skip non-existent values:
```suggestion
            if (contentType.HistoryCleanup != null)
            {
                xml.Add(SerializeCleanupPolicy(contentType.HistoryCleanup));
            }
```

Reply:
dcbd51cf